source/partials/drupal-8-convert-to-composer.md

```diff
@@ -11,7 +11,7 @@ To minimize issues, these steps make the codebase changes in a new branch:
 1. Add the Pantheon Drupal Project upstream as a new remote called `ic`, fetch the `ic` upstream, and checkout to a new local branch based on it called `composerify`:
 
   ```bash{outputLines:2}
-  git remote add ic git@github.com:pantheon-upstreams/drupal-recommended.git && git fetch ic && git checkout --no-track -b composerify ic/master
+  git remote add ic git@github.com:pantheon-upstreams/drupal-composer-managed.git && git fetch ic && git checkout --no-track -b composerify ic/master
```

Comment:
Another `ic/main`

Reply:
Addressed in the previous commit [979fe19](https://github.com/pantheon-systems/documentation/pull/7292/commits/979fe19a942a7302db1f51cdee3cd8cd4a3e6683)